### ISSUE TITLE
Add the option of Invite Code (optional)

### ISSUE THREAD

**Issue by ahrasis** [ReFlar/doorman#9]
As requested by Faaju at [Doorman thread](https://discuss.flarum.org/d/17845-doorman-by-reflar/7) in Flarum Discuss:
```
Could you add the option of Invite Code (optional)?
What would it look like?

After selecting this option in the admin panel, the user could leave the code field blank during registration and register without entering the code.
```

**Comment by iPurpl3x** [comment]
I need this too, so I will make a fork and implement this myself.

**Comment by CDK2020** [comment]
Closed in #12
